platform/accessibility/common-issues/pdf/no-structural-tags.md

```diff
@@ -0,0 +1,101 @@
+# No structural tags
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+ - [Statement of the issue](#statement-of-the-issue)
+ - [Definition of done](#definition-of-done)
+ - [Tag your document](#tag-your-document)
+    - [Define headings in Microsoft Word](#define-headings-in-microsoft-word)
+    - [Define lists in Microsoft Word](#define-lists-in-microsoft-word)
+    - [Export a tagged PDF](#export-a-tagged-pdf)
+ - [Test for compliance](#test-for-compliance)
+    - [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+    - [Adobe Acrobat Pro accessibility tools](#adobe-acrobat-pro-accessibility-tools)
+  - [Learn more](#learn-more)
+
+## Statement of the issue
+
+- **Synopsis:** PDFs (portable document files) must have tagged headers, tables, lists, and paragraphs. Assistive technology can navigate to sections of a document using these tags. PDFs without tags require users to read the document instead of skimming for what they want.
+- **Severity:** Serious
```

Comment:
On the "Table is missing headers" doc you used the defect severity level rating. Is there a reason you addressed this one differently? (As a sidenote, the link to the severity rating doc is broken on the "Table is missing headers" doc.)